test/gtest/aca_test_zeta_programming.cpp

```diff
@@ -142,3 +172,21 @@ TEST(zeta_programming_test_cases, DISABLED_auxgateway_test)
 
   EXPECT_EQ(retcode, EXIT_SUCCESS);
 }
+
+TEST(zeta_programming_test_cases, DISABLED_zeta_gateway_path_CHILD)
+{
+  // TODO: The relative path of the CHILD configuration file
+  string zeta_gateway_path_CHILD_config_file = "./...";
+  aca_test_zeta_setup(zeta_gateway_path_CHILD_config_file);
+
+  // do some validate
+}
+
+TEST(zeta_programming_test_cases, DISABLED_zeta_gateway_path_PARENT)
+{
+  // TODO: The relative path of the PARENT configuration file
+  string zeta_gateway_path_PARENT_config_file = "./...";
+  aca_test_zeta_setup(zeta_gateway_path_PARENT_config_file);
+
+  // do some validate
```

Comment:
@PikaPikaW please implement the testcases, thank you.

Reply:
on parent side, please check if the expected rules are installed, and then try to do a ping from parent port to child port, confirm the ping is successful.